lib/workers/repository/config-migration/branch/migrated-data.ts

```diff
@@ -0,0 +1,86 @@
+import detectIndent from 'detect-indent';
+import JSON5 from 'json5';
+import { migrateConfig } from '../../../../config/migration';
+import { logger } from '../../../../logger';
+import { readLocalFile } from '../../../../util/fs';
+import { detectRepoFileConfig } from '../../init/merge';
+
+export class MigratedData {
+  constructor(
+    private readonly migratedContent: string,
+    private readonly configFileName: string
+  ) {}
+
+  get content(): string {
+    return this.migratedContent;
+  }
+
+  get fileName(): string {
+    return this.configFileName;
+  }
+}
```

Comment:
can we use a simple interface? i don't see any benefit of this class 🤔

Reply:
changed to an interface